docs/classic-ui/images.md

```diff
@@ -285,18 +285,26 @@ To access image scales, which are normally not accessible to the current user, o
 
 ## Responsive image support
 
-Plone supports the generation of picture tags with `srcset`s for image optimization.
+Plone supports the generation of [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) tags with `srcset`s for image optimization.
 Additionally, you can define [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries) for [art direction](classic-ui-images-responsive-image-support-art-direction) and further optimization.
 
-The configuration allows you to define different picture variants, such as `Large`, `Medium`, or `Small`.
+The configuration allows you to define different `picture` variants, such as `Large`, `Medium`, or `Small`.
 Users can choose from them in editors, such as TinyMCE, and developers can use them in templates.
 
+To generate a `picture` tag, use the following code.
+
+```python
+from plone import api
+
+scale_util = api.content.get_view("images", context, request)
+tag = scale_util.picture("image", scale="larger", picture_variant="large")
+```
 
```

Comment:
Maybe we can add the template implementation here too?

snippet:

```
<div tal:define="scale_view context/@@images">
  <img tal:replace="structure python:scale_view.picture('image', scale='larger', picture_variant='large'" />
</div>
```

Reply:
Added!

Reply:
Can be useful to remember that this can be done on brains too? Just pointing to `plone/app/contenttypes/browser/templates/listing_summary.pt` as an example. 

In general, the receipt for rendering images html code is "get the @@images view on the context you need (object or brain) and use `picture' or `tag` method of the view, using the object as the first parameter if the view has not been already took using it as context". This is my understanding on how it works.

Do tag support all the img tag attributes? In some particular cases, a query to the catalog can retrieve all the info about the original image and let people create a custom html snippet.